OneSignalSDK/unittest/src/test/java/com/onesignal/ShadowOSUtils.java

```diff
@@ -72,14 +78,14 @@ public static void resetStatics() {
       carrierName = "test1";
       subscribableStatus = 1;
 
+      isRunningOnMainThread = false;
```

Comment:
We would like to have new `isRunningOnMainThread` check fail in tests when it should, so we know if something broke when travisCI runs.

Also this line is dangerous as it will hide other future checks we add to other things.

Reply:
Which tests fail if we don't have this line?